<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsultaAluno</class>
 <widget class="QWidget" name="ConsultaAluno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consultar Aluno</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/recursos/Imagens/pesquisar.png</normaloff>:/recursos/Imagens/pesquisar.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>1121</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,2,1">
      <item row="0" column="0">
       <widget class="QLabel" name="labelProcurar">
        <property name="text">
         <string>Procurar:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioMatricula">
        <property name="text">
         <string>Matricula</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioCpf">
        <property name="text">
         <string>Cpf</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="radioNome">
        <property name="text">
         <string>Nome</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="btnConsultar">
        <property name="text">
         <string>Consultar</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEditSearch">
        <property name="accessibleName">
         <string/>
        </property>
        <property name="maxLength">
         <number>50</number>
        </property>
        <property name="placeholderText">
         <string>Digite aqui</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableView">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditSearch</tabstop>
  <tabstop>btnConsultar</tabstop>
  <tabstop>radioMatricula</tabstop>
  <tabstop>radioCpf</tabstop>
  <tabstop>radioNome</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
